ts/src/transports/msStream.ts

```diff
@@ -0,0 +1,60 @@
+import {BrowserHeaders} from "browser-headers";
```

Comment:
Can you add a comment to this whole file as to why this magic is needed + add the doc links to where you found these incantations? :)